BridgeAppSDK/SBABridgeInfo.swift

```diff
@@ -80,6 +80,11 @@ public protocol SBABridgeInfo: class {
      * Mapping of task identifier and associated info for creating a task
      */
     var taskMap: [NSDictionary]? { get }
+    
+    /**
+     * Name of .pem certificate file to use for uploading to Bridge (without the .pem extension)
+     */
+    var certificateName: String? { get }
```

Comment:
If this is defined as optional then should have a drop-through for the encryption.